src/components/structures/UserSettings.js

```diff
@@ -60,10 +61,21 @@ module.exports = React.createClass({
             threePids: [],
             phase: "UserSettings.LOADING", // LOADING, DISPLAY
             email_add_pending: false,
+            vectorVersion: null,
         };
     },
 
     componentWillMount: function() {
+        if (PlatformPeg.get()) {
+            PlatformPeg.get().getAppVersion().done((appVersion) => {
```

Comment:
is there a situation in which getAppVersion thows an exception, and takes out react? worth sticking a `q().then` in front of this?